styles.js

```diff
@@ -1,5 +1,46 @@
 import { createGlobalStyle } from "styled-components";
 
+export const theme = {
+  primaryColor: "#efe8d8",
+  textColor: "#262524",
+  secondaryColors: {
+    sports: "#86b2b1",
+    culture: "#c16b6b",
+    food: "#e2aa60",
+    outdoor: "#96ad8c",
+    default: "#efe8d8",
+  },
+  fonts: {
+    heading: "Averia Serif Libre, serif",
+    text: "Noto Serif Thai Condensed, serif",
+  },
+  fontSizes: {
+    small: "18px",
+    medium: "24px",
+    large: "32px",
```

Comment:
using rem or a similar unit would probably be better here so users can tweak the font sizes to their needs.